Supernova.Enum.Generators/Extensions/NamespaceSymbolExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using Microsoft.CodeAnalysis;
+
+namespace Supernova.Enum.Generators.Extensions;
+
+/// <summary>
+/// Provides extension methods for <see cref="INamespaceSymbol"/> objects.
+/// </summary>
+public static class NamespaceSymbolExtensions
+{
+    /// <summary>
+    /// Gets the full name of the namespace, including parent namespaces.
+    /// </summary>
+    /// <param name="namespaceSymbol">The namespace symbol.</param>
+    /// <param name="fullName">Optional. The initial full name to start with.</param>
+    /// <returns>The full name of the namespace.</returns>
+    public static string FullNamespace(this INamespaceSymbol namespaceSymbol, string fullName = null)
```

Comment:
this method exist in to SymbolExtensions

Reply:
The original FullNamespace extension in the SymbolExtensions file is replaced by the new extension file. The why I already explained in the other comment.